packages/zstd/zstd.0.4/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+maintainer: "ygrek@autistici.org"
+homepage: "https://github.com/ygrek/ocaml-zstd"
+license: "BSD-3-Clause"
+authors: [ "ygrek" ]
+tags: [ "org:ygrek" "clib:zstd" ]
+doc: [ "https://ygrek.org/p/ocaml-zstd/api/index.html" ]
+dev-repo: "git+https://github.com/ygrek/ocaml-zstd.git"
+bug-reports: "https://github.com/ygrek/ocaml-zstd/issues"
+build: [
+  ["dune" "subst"] {dev}
+  ["dune" "build" "-p" name "-j" jobs
+    "@install"
+    "@doc" {with-doc}
+    "@runtest" {with-test}]
+]
+depends: [
+  "ocaml"
+  "dune" {>= "3.7"}
+  "ctypes"
```

Comment:
I think you need to constrain the `ctypes version more here - otherwise this fails with various errors (see for instance this [lower-bounds check](https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/c65e42c54f2a005acf0e197bf425395984d5c727/variant/compilers,4.03,zstd.0.4,lower-bounds))